mod-tags/src/test/resources/domain/tags/features/tags.feature

```diff
@@ -10,8 +10,45 @@ Feature: Mod-tags integration tests
     * configure headers = headersUser
     * def tag = read('classpath:samples/tag.json')
 
-  Scenario: Test GET collection of tags
-    Given path 'tags'
-    When method GET
-    Then status 200
-    And match response.totalRecords == 2
\ No newline at end of file
+ # ================= positive test cases =================
+  @Undefined
+  Scenario: GET '/tags' should return 200 and collection of tags
+    * print 'undefined'
+
+  @Undefined
+  Scenario: GET '/tags/{id}' should return 200 and tag with specified id
+    * print 'undefined'
+
+  @Undefined
+  Scenario: POST '/tags' should return 201 and created tag with id
+    * print 'undefined'
+
+  @Undefined
+  Scenario: PUT '/tags/{id}' should return 204 if tag was successfully updated
+    * print 'undefined'
+
+  @Undefined
+  Scenario: DELETE '/tags/{id}' should return 204 if tag was successfully deleted
+    * print 'undefined'
+
+
+  # ================= negative test cases =================
+  @Undefined
+  Scenario: ALL TYPES '/tags' should return 400 when malformed request body or query parameter
+    * print 'undefined'
+
+  @Undefined
+  Scenario: ALL TYPES '/tags' should return 401 when user not authorized to perform action
+    * print 'undefined'
+
+  @Undefined
+  Scenario: GET, PUT, DELETE '/tags/{id}' should return 404 if tag with specified id not found
```

Comment:
Please create scenario for each HTTP method

Reply:
Updated